### DISCUSSION TITLE
Abrir conta bancária na Europa

### DISCUSSION THREAD

**Discussion by gabrweu** [DyegoCosta/trabalhando-remoto#186] [General]
E aí, pessoal.

Minha situação é parecida com a de vários aqui: consegui um emprego remoto para um empresa Europeia, e gostaria de não converter todo meu salário para BRL. Reter uma parte numa conta em Euros me ajudaria em futuras viagens ou uma eventual mudança para fora, além de abrir uma nova porta para investimentos lá.

Entretanto, as opções me parecem bem limitadas.

Recebo o pagamento pela TransferWise, e criei uma Broderless Account para manter o dinheiro em Euros provisoriamente, e fazer o processo de transferir para minha conta no Brasil eu mesmo (no lugar de receber direto lá pela empresa). Manter quantidades elevadas lá de forma definitiva não me parece uma boa ideia, afinal, não são um banco, e a conta não é verdadeiramente uma conta bancária _minha_.

Pelo que pesquisei, isso me deixa apenas com opções de bancos que requerem comprovante de residência, ou que simplesmente eu tenha um endereço na Europa. 

Muita gente (anecdotally) usa o N26 daqui do Brasil sem ter pisado na Europa, enviando o cartão (necessário para ativar a conta) para parentes ou usando serviços de Mail Forwarding. É uma possibilidade, e inclusive tenho parentes/conhecidos tanto na Irlanda quando na Alemanha para fazer esse processo por mim. 

Mas, isso é claramente contra como esperam que tu use o serviço, e imagino que constitua uma violação dos termos de uso. Seria relativamente fácil identificar que o usuário não mora no endereço cadastrado, e daria um motivo para fecharem a conta. Fico com o pé atrás ainda mais depois de ler sobre alguns casos em que houve esse fechamento de conta, além de sobre algumas restrições que a empresa colocou sobre Brasileiros alguns anos atrás (não mais vigentes, pelo que li agora).

Tem também um limite de 18 meses que o usuário pode ficar fora da Europa sem dar satisfação para o banco. No meu caso, iria para a Europa pelo emprego pelo menos duas vezes por ano, e poderia me enquadrar nessa regra, mas, considerando a situação atual, Corona e tudo mais, fica difícil de saber quando seria a primeira viagem.

Há também um banco virtual Português, ActivoBank, e o Revolut. Os dois também pedem que o usuário more na Europa, e precisariam desse esquema de Mail Forwarding, constituindo opções não muito confiáveis pelo mesmo motivo do N26.

Então, essa issue serve mais para ver se algum de vocês conhece alguma opção usável daqui do Brasil, ou se tem experiência com esses bancos acima morando daqui. Fica aberta a discussões e sugestões sobre.

Temos a issue #93 falando desse mesmo assunto mas para Dólares/o Canadá, e a #64 falando de bancos nos EUA.

**Comment by DyegoCosta** [comment]
@gabrielbunselmeyer Não sei dizer quanto aos bancos europeus, mas bancos nos EUA você precisa de um endereço local (de lá) e estar lá presencialmente para a abrir a conta mas logo em seguida consegue colocar seu endereço brasileiro, para cobrança e/ou correspondência.

Bem possível que já tenha visto isso, mas dei uma pesquisada aqui e achei esse artigo que parece explicar bem as opções na europa: https://transferwise.com/us/blog/european-bank-account-for-non-residents

**Comment by gabrweu** [comment]
@DyegoCosta Pois é, tinha dado uma lida nesse artigo, mas todas as opções citadas precisam que eu esteja lá, ou tenha comprovante de residência. Acho que não tem muito como fugir disso mesmo. 

Talvez a melhor opção seja abrir uma conta quando viajar para lá, que não sei quando vou ter uma oportunidade. O N26 parece ser a melhor alternativa de banco, mas não tô muito confortável com a ideia desse mail forwarding. 

Se alguém conhecer algum banco mágico que atenda o meu caso, aceito sugestões. Vi um Suíço que abre conta pela internet sem muitos problemas, e parecia me atender perfeitamente, mas custava 5 mil Euros o processo hehehe. 

Provisoriamente parece que vou ter que ir usando a Borderless Account da TransferWise mesmo.

**Comment by lucassimao** [comment]
Cara, se for uma situação de longo prazo você pode cogitar a possibilidade da cidadania eletrônica da estônia, depois da uma olhada https://e-resident.gov.ee/

**Comment by gabrweu** [comment]
@lucassimao Faz uns dias que vi que era uma opção, mas descartei por achar que era algo bem mais complexo do que o necessário. Hoje de noite vou dar mais uma olhada e reavaliar. Só não faço ideia de como funcionaria criar uma empresa lá E ter um CNPJ aqui, pensando em carga tributária e afins.

Parece que é tranquilo abrir uma conta para empresas com apenas 1 pessoa, ainda mais de TI, tendo a e-residência.

Valeu pela sugestão!

**Comment by samiraguiar** [comment]
@gabrielbunselmeyer eu tenho as mesmas necessidades que vc e uso a Payoneer. Como o recebimento é em euro e não dólar, o spread é levemente menor (2%). Peguei o cartão deles e uso pra fazer todo tipo de compra que posso pagar diretamente em euro, inclusive estadias em hotéis e Airbnbs dentro do Brasil. Assim já economiza o que gastaria com o spread se tivesse transferido o dinheiro pra cá. Outra vantagem é que dá pra sacar o dinheiro em outras moedas quando estiver em viagem, e dependendo do valor do saque fica mais barato do que pagar a conversão Euro -> Real + Real -> Outra Moeda (cotação turismo).

A maior desvantagem disso é a anuidade do cartão de 29 dólares.

Se encontrar outros serviços nesse modelo que vc disse, atualiza aqui por favor.

**Comment by gabrweu** [comment]
@samiraguiar Vou ir atualizando aqui sim. Até porque o assunto da issue pode mudar bastante com o tempo, então seria uma boa atualizar com qualquer questão relevante que for aparecendo.

Pelo que entendi, o Payoneer também só te dá acesso a uma "conta virtual" como serviço, da mesma forma que a Borderless Account da TransferWise, não? Meu receio de usar esse tipo de coisa, e não uma conta verdadeira, em um banco, no meu nome, é a possível falta de segurança.

Mas boas informações aí, vou levar em consideração se a empresa apresentar flexibilidade do serviço de envio do dinheiro.

**Comment by samiraguiar** [comment]
@gabrielbunselmeyer Não, não é uma conta no sentido tradicional. Não tem poupança ou investimento, e não deixo muito dinheiro lá. Costumo deixar mais quando vou viajar, pra chegar no outro país e sacar, quando vou fazer alguma compra ou quando não quero sacar tudo porque acho que o euro vai subir.
Já procurou alguma corretora da Europa? Sei que pros EUA tem corretora que te deixa abrir conta direto daqui, talvez lá tenha também. Se tiver, uma ideia é ir jogando pra corretora e já investir.

**Comment by gabrweu** [comment]
@samiraguiar Pois é, minha ideia principal é guardar uma grana pra quando for viajar. Investimentos viriam depois, e jogar direto pra corretora talvez me tire essa primeira parte. 

Pelo que já pesquisei, maioria das corretoras só te deixa abrir conta se tiver uma conta de banco real mesmo, esses esquemas estilo Payoneer e TransferWise não devem dar certo. Fim de semana dou uma pesquisada e atualizo aqui.

**Comment by aracyla** [comment]
Eu achei um artigo comentando que tem alguns bancos alemães que deixam abrir conta com uma série de restrições.
https://www.firma.de/en/company-formation/opening-a-german-bank-account-as-a-non-german-citizen-how-it-works/

* Eu tenho uma conta lá que abri quando fiz intercambio, eu deixei um dinheiro lá e ainda tá cobrindo a anuidade/mensalidade. Vou tentar entrar em contato com os caras pra saber da possibilidade de manter pelo menos a conta ativa como savings account e dou um feedback depois. (btw o banco é o Sparkasse)

**Comment by gabrweu** [comment]
O banco Millenium de Portugal aparentemente não tem restrição alguma, e pode se abrir uma conta sem sair do Brasil. Único requerimento é um depósito inicial de 500 Euros e uma visita à agência deles em São Paulo.

Me parece uma boa pós-Corona, mas não pesquisei tão a fundo ainda.

**Comment by hails** [comment]
@gabrielbunselmeyer fora o Millenium, existe também o banco Atlantico em Portugal, que voce consegue abrir uma conta de EUR e USD usando seus documentos aqui do Brasil.
https://www.atlantico.eu/

**Comment by gabrweu** [comment]
@hails Boa, valeu! Se tiver alguma dica/recomendação sobre, ou recomendação entre os dois, sou todo ouvidos.

Vi que tem bastante gente reclamando do app do Millenium e de alguns tipos de transação não funcionarem direito daqui do BR. Vou dar uma analisada eventualmente e posto aqui pros interessados, questões de taxas, cobranças e afins. Quero dar uma pesquisada sobre segurança do dinheiro nesses bancos também.

**Comment by alvesmog** [comment]
Também estou em busca de alternativas para manter o saldo recebido em euros. Já vi recomendarem o banco atlantico europa, mas pelo que entendi, eles não receberiam de conta PJ no exterior, então de toda forma teria que converter o dinheiro para BRL e depois mandar dnv pra lá. Não me parece muito vantajoso.

**Comment by gabrweu** [comment]
@alvesmog se tu recebe como PJ, não tem muito como escapar da RemessaOnline.

Tive conta no Atlantico Europa por um tempo mas não senti muita firmeza no banco em si, e fechei a conta quanto introduziram algumas taxas. E AFAIK não tem como receber por lá de forma correta como PJ. 

Atualmente recebo pela RemessaOnline (PJ) e converto de volta ao enviar para uma corretora de fora. Não é o ideal, mas é o único jeito que encontrei se for fazer tudo certinho.

**Comment by PatrickRNG** [comment]
@gabrielbunselmeyer 
Estou nessa mesma situação, uso o RemessaOnline e estou querendo guardar dinheiro em Euro, a opção é essa ou usar a Payoneer.

Com a Payoneer, eu poderia receber como PJ, e deixar o dinheiro que quero guardar na conta e o resto transferir pra conta PJ. O problema que não sei, é se caso eu for viajar para a Europa, eu não sei se eu poderia gastar dinheiro da Payoneer com o cartão deles, já que tecnicamente seria dinheiro da minha empresa.
Como eu poderia fazer isso? Na Europa eu abro uma conta de banco local e transfiro da Payoneer para a conta local? (EUR -> EUR da €1.50 de taxa) E na contabilidade isso seria antecipação de lucros?

Ainda estou em dúvida sobre qual das duas opções seria a melhor.
A Payoneer como eu comentei acima.
Ou da RemessaOnline/Husky para PJ -> PF -> Corretora (como a Wise).

Edit: Optei pela Husky -> PJ -> PF -> Wise. Justamente pela contabilidade, eu compro a moeda pela PF mesmo, sem preocupação futuramente.

**Comment by dcecchini** [comment]
Deixando aqui um banco que eu utilizei um pouco e que me parece bem bacana. A Wise é muito boa também, mas se você quer algo que seja de sangue BR e pé na Europa, eu conheci o Capitual (https://www.capitual.com/). Usei para brincar com criptomoedas, mas tem saldo em diversas moedas e transferências pelo sistema europeu era gratuito. 

Deixo como alternativa,
